web/src/containers/pages/SignUpPage.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC, useState } from 'react'
+import { Col } from 'react-grid-system'
+import Header, { HeaderNavigation } from 'src/components/molecules/Header'
+import SignUp from 'src/containers/organisms/SignUp'
+
+const LogInPage: FC = () => {
```

Comment:
ファイル名と中身が違ってるので下記の感じですかね

```suggestion
const SignUpPage: FC = () => {
```